modules/openy_features/openy_node/modules/openy_node_alert/src/Service/AlertManager.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Drupal\openy_node_alert\Service;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+
+/**
+ * Class AlertManager.
+ *
+ * @package Drupal\openy_node_alert\Service
+ */
+class AlertManager {
+
+  /**
+   * The alerts array.
+   *
+   * @var \Drupal\openy_node_alert\Service\AlertBuilderInterface[]
+   */
+  protected $alerts = [];
+
+  /**
+   * An array with sorted alerts by priority, NULL otherwise.
+   *
+   * @var null|array
+   */
+  protected $alertsSorted = NULL;
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * Constructs the LandingPageAlertBuilder.
+   *
+   * @param EntityTypeManagerInterface $entity_type_manager
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->nodeStorage = $entity_type_manager->getStorage('node');
+  }
+
+  /**
+   * Adds alerts to internal service storage.
```

Comment:
I don't think this is the correct description. It collects alert builders, not alerts.